src/test/resources/movie-tests.adoc

```diff
@@ -857,6 +859,107 @@ RETURN  movie  {  .title  } AS  movie
 ORDER BY movie.year DESC LIMIT $movieFirst
 ----
 
+=== Deeply nested orderBy
+
+.GraphQL-Query
+[source,graphql]
+----
+{
+  Movie(orderBy:title_desc) {
+    title
+    actors(orderBy:name_desc) {
+      name
+      movies(orderBy:[title_asc, title_desc]) {
+        title
+      }
+    }
+  }
+}
+----
+
+.Cypher Params
+[source,json]
+----
+{}
+----
+
+.Cypher
+[source,cypher]
+----
+MATCH (movie:Movie)
+RETURN movie {
+  .title,
+  actors: apoc.coll.sortMulti([(movie)<-[:ACTED_IN]-(movieActors:Actor) | movieActors {
+    .name,
+    movies: apoc.coll.sortMulti([(movieActors)-[:ACTED_IN]->(movieActorsMovies:Movie) | movieActorsMovies {
+      .title
+    }], ['^title','title'])
+  }], ['name'])
+} AS movie
+ORDER BY movie.title DESC
+----
+
+=== Deeply nested query using temporal orderBy
+
+.GraphQL-Query
+[source,graphql]
+----
+{
+  User(orderBy: born_desc){
+    name
+    born {formatted}
+    knows(orderBy: born_asc) {
+      name
+      born {formatted}
+      knows(first: 2, offset: 1, orderBy: [born_desc, born_desc]) {
+        name
+        born {formatted}
+      }
+    }
+  }
+}
+----
+
+.Cypher Params
+[source,json]
+----
+{
+  "userKnowsKnowsOffset": 1,
+  "userKnowsKnowsFirst": 2
+}
+----
+
+.Cypher
+[source,cypher]
+----
+MATCH (user:User)
+RETURN user {
+  .name,
+  born: {
+    formatted: toString(user.born)
+  },
+  knows: [sortedElement IN apoc.coll.sortMulti([(user)-[:KNOWS]->(userKnows:User) | userKnows {
+    .name,
+    born: userKnows.born,
+    knows: [sortedElement IN apoc.coll.sortMulti([(userKnows)-[:KNOWS]->(userKnowsKnows:User) | userKnowsKnows {
+      .name,
+      born: userKnowsKnows.born
+    }], ['born','born']) | sortedElement {
```

Comment:
should one of these be ascending?

Reply:
do we need to differentiate between nested levels?

Reply:
Do you mean to have different names?